ENGAGEHF/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+//  DashboardView.swift
+//  ENGAGEHF
+//
+//  Created by Nick Riedman on 4/3/24.
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct Greeting: View {
+    @State private var dateString: String?
+    
+    var body: some View {
+        HStack {
+            Text("Hello, world!")
+                .font(.title2)
+                .accessibilityLabel(Text("DASHBOARD_GREETING"))
+            Spacer()
+            Text(dateString ?? "No date")
+                .font(.title2)
+                .accessibilityLabel(Text("DASHBOARD_DATE"))
+        }
+        .padding()
+        .task {
+            getDateString()
+        }
+    }
+    
+    private func getDateString() {
+        let currentDate = Date()
+        let dateFormatter = DateFormatter()
+        
+        dateFormatter.dateStyle = .short
+        
+        dateString = dateFormatter.string(from: currentDate)
+    }
+}
+
+struct Dashboard: View {
+    @Binding var presentingAccount: Bool
+    
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Greeting()
+                Spacer()
+            }
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationTitle("ENGAGE-HF: Home")  // Todo: Make this white
+            
+            .toolbar {
+                ToolbarItem(placement: .topBarTrailing) {
+                    AccountButton(isPresented: $presentingAccount)
+                        .foregroundColor(.white)
+                        .accessibilityLabel(Text("DASHBOARD_ACC_BTN"))
+                }
+            }
+            .toolbarBackground(.visible, for: .navigationBar)
+            .toolbarBackground(Color("AccentColor"), for: .navigationBar)
```

Comment:
I am not a huge fan of trying to customize the Toolbar with solid non-standard colors. Brings a lot of complexities down the line. I would honestly suggest to stick with the Apple Default style here and rather style some UI elements using the accent color (which they are done automatically, e.g. the account button).

```suggestion
```